middleware/setCurrentCountry.js

```diff
@@ -0,0 +1,12 @@
+module.exports = (req, res, next) => {
+  const originalUrl = req.originalUrl;
+  const directories = originalUrl.split('/');
+
+  if (directories.length > 2) {
+    res.locals.currentCountry = directories[1].toLowerCase();
```

Comment:
do we want or need to do some kind of whitelist here? (match against a list of countries that we're actually set up for?)

Reply:
I pretty much copy/pasted this from our FE country lookup. Do you think it's necessary to gracefully handle this? I suppose we'd just fallback to the US as the default like if there isn't a directory? At this point I think the only way this could happen is if someone manually changed the url so I'm not super worried about it

Reply:
fair point, doesn't do much for us at this stage